notebooks/custom_kinfraglib/1_2_custom_filters_drug_likeness.ipynb

```diff
@@ -53,15 +53,23 @@
   },
```

Comment:
<p>can you re-run this notebook? It seems to contain an import error here :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/KinFragLib/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>